download_file_from_bucket.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import os
+
+from google.cloud import storage
+
+
+def load_bucket_sample_file(sample_file):
+    client = storage.Client()
+
+    bucket = client.get_bucket(os.getenv('SAMPLE_BUCKET'))
+    blob = storage.Blob(sample_file, bucket)
+
+    with open(os.path.join("sample_files", sample_file), 'wb+') as file_obj:
+        client.download_blob_to_file(blob, file_obj)
+
+    print(f'downloaded file {sample_file} from gcp bucket {os.getenv("SAMPLE_BUCKET")}, now loading')
```

Comment:
I would drop the "now loading" part, in case we use the script in a standalone way - it could cause alarm to somebody if they thought that it was starting the sample load!